### PR TITLE
First draft of footer

### PR DESCRIPTION
**Pull request by Yaomimi**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

* [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
* [ ] I have checked the dependencies
* [ ] I have given the fractal status “ready” to my component
* [ ] I have declared `@define mycomponent` in the SCSS file
* [ ] I have specified `margin: 0;` on the CSS component
* [ ] I have provided tests
* [ ] I follow the naming guidelines
* [ ] the component supports composition
* [ ] there are no hardcoded strings (all content come from the context)
* [ ] I have filled the README.md file (at least a few lines)
* [ ] My component is listed in the root README
* [ ] My PR has the right label(s)

### CODE REVIEW

File: src/systems/ec/specs/components/footer/docs/usage.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
 ## Usage
 
-Do's and don'ts
+The footer is located at the bootom of a page. It provides inforamtion from the European Commission. 
+
+### Main content area
+
+The main content area of the footer offers access to the political and the informational section of the European Commission. It also contain links to the European Union and the EU institutions.
+Users can also find information of copyright, the Commission's new web presence, resources for partners, cookies, legal notices, contact and social media.
```
Comment (yhuard) [inline src/systems/ec/specs/components/footer/docs/usage.md]: `information of` or `information about`?

File: src/systems/ec/specs/components/footer/docs/usage.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+
+### Custom area
+
+In some circumstances, a custom are can be added if the main and secondary content areas do not satisfy the requirements.  A custom area can contain:
```
Comment (yhuard) [inline src/systems/ec/specs/components/footer/docs/usage.md]: `a custom are can` <- `area` I guess

File: src/systems/ec/specs/components/footer/docs/usage.md [hunk 3]
```diff
@@ -1,3 +1,22 @@
...
+
+* idntification of the site name
```
Comment (yhuard) [inline src/systems/ec/specs/components/footer/docs/usage.md]: `idntification` -> `identification`

File: src/systems/ec/specs/components/footer/docs/usage.md [hunk 4]
```diff
@@ -1,3 +1,22 @@
...
+* iformation on how to connect to social platforms
```
Comment (yhuard) [inline src/systems/ec/specs/components/footer/docs/usage.md]: `iformation` -> `information`

**Review by yhuard — CHANGES_REQUESTED** [review]
Hi @Yaomimi 

Thank you for your PR!

There are some typos here and there. You can correct them from the "Files changed" tab and then click on the pencil icon. Maybe has @degliwe already shown you how to do this.

File: src/systems/ec/specs/components/footer/docs/usage.md (hunk 1 above)
Comment (Yaomimi) [reply src/systems/ec/specs/components/footer/docs/usage.md]: Hi Yannick, thanks for checking...grammarly doesn't work here. I will go through it again.

File: src/systems/ec/specs/components/footer/docs/usage.md (hunk 4 above)
Comment (Yaomimi) [inline src/systems/ec/specs/components/footer/docs/usage.md]: I also realize that when I try to edit the sentence then the text afterward begin to disappear. This is why I have missing letters of the words...need to still get used to it. (That wasn't so smart in my view, the way it worked, or I just haven't figured out how to do it.)

**Pull request closed without merging**